platforms/core-runtime/build-configuration/src/integTest/groovy/org/gradle/interal/buildconfiguration/tasks/UpdateDaemonJvmIntegrationTest.groovy

```diff
@@ -21,13 +21,13 @@ import org.gradle.api.JavaVersion
 import org.gradle.buildconfiguration.tasks.UpdateDaemonJvm
 import org.gradle.integtests.fixtures.AbstractIntegrationSpec
 import org.gradle.integtests.fixtures.AvailableJavaHomes
-import org.gradle.internal.buildconfiguration.BuildPropertiesDefaults
-import org.gradle.internal.buildconfiguration.fixture.BuildPropertiesFixture
+import org.gradle.internal.buildconfiguration.DaemonJVMPropertiesDefaults
+import org.gradle.internal.buildconfiguration.fixture.DaemonJVMPropertiesFixture
 import org.gradle.internal.jvm.Jvm
 import org.gradle.test.precondition.Requires
 import org.gradle.test.preconditions.IntegTestPreconditions
 
-class UpdateDaemonJvmIntegrationTest extends AbstractIntegrationSpec implements BuildPropertiesFixture {
+class UpdateDaemonJvmIntegrationTest extends AbstractIntegrationSpec implements DaemonJVMPropertiesFixture {
```

Comment:
First this file should be considered to be named as `UpdateDaemonJVMIntegrationTest` keeping aligned with the renaming of other classes which uses the JVM with capital letters. This also applies to `UpdateDaemonJvm.java` and `UpdateDaemonJvmModifierTest.groovy`

However, this change goes against the naming already used for the existing classes. IMHO we should consider keeping everything as `Jvm` for consistency reasons 

<img width="481" alt="Screenshot 2024-04-24 at 15 04 14" src="https://github.com/gradle/gradle/assets/18151158/bc237c56-2698-49cb-99f1-d8937702c29e">

Reply:
That's a very good point and I hesitated on it, without finding a way to answer my hesitation. Thanks for pointing these out!

Reply:
Aligned to use `Jvm` everywhere